gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/TranslateSchemasTask.java

```diff
@@ -44,7 +44,7 @@ public void translate()
     getProject().getLogger().info("Translating data schemas ...");
     _destinationDir.mkdirs();
 
-    String resolverPathStr = _resolverPath.plus(getProject().files(_inputDir)).getAsPath();
+    String resolverPathStr = getProject().files(_inputDir).plus(_resolverPath).getAsPath();
```

Comment:
You might want to document why this order matters.

Reply:
Done